src/services/taxonomyService.ts

```diff
@@ -0,0 +1,34 @@
+// import { BackofficeApi } from '../api/BackofficeClient';
+import { toError } from 'fp-ts/lib/Either';
+import {storageTokenOps} from '@pagopa/selfcare-common-frontend/utils/storage';
+import { ENV } from '../utils/env';
+import { Taxonomies } from '../api/generated/portal/Taxonomies';
+import { getTaxonomies as getTaxonomiesMock } from './__mocks__/taxonomyService';
+
+
+
+// TODO FIX GENERATED CLIENT AND RE-INTRODUCE BACKOFFICE METHOD
```

Comment:
👀

Reply:
The service will be reworked with the new APIs for the taxonomies (that have yet to be defined)